javaCarAPI/src/main/java/api/CarAPI.java

```diff
@@ -0,0 +1,261 @@
+package api;
+
+import api.sensor.Infrared;
+import api.sensor.Odometer;
+import javafx.scene.image.Image;
+import javafx.scene.image.PixelWriter;
+import javafx.scene.image.WritableImage;
+import javafx.scene.paint.Color;
+import org.eclipse.paho.client.mqttv3.*;
+import org.eclipse.paho.client.mqttv3.persist.MqttDefaultFilePersistence;
+import java.nio.charset.StandardCharsets;
+import java.util.*;
+import java.util.function.Consumer;
+
+public class CarAPI {
+
+    private static final int MOVEMENT_SPEED = 70;
+    private static final int STRAIGHT_ANGLE = 0;
+    private static final int STEERING_ANGLE = 50;
+    private static final int IDLE_SPEED = 0;
+    private static final int QOS = 1;
+    private static final int IMAGE_WIDTH = 320;
+    private static final int IMAGE_HEIGHT = 240;
+
+    private String throttleControl;
```

Comment:
Please change the name of the mqtt-topic variables to something that conveys that they are, in fact, related to mqtt topics and not something else.